README.md

```diff
@@ -82,6 +84,42 @@ along with an access token in `~/.eslint_d`.
 
 If you're using `eslint_d` in any other editor, please tell me!
 
+### Automatic Fixing
+
+`eslint_d` has an additional flag that `eslint` does not have, `--fix-to-stdout`
+which prints the fixed file to stdout. This allows editors to add before save
+hooks to automatically fix a file prior to saving. It must be used with
+`--stdin`.
+
+- __Emacs__: Add this to your `init.el`, be sure to add hooks for the actual
+  major modes you use:
+
+    ```elisp
+    (defun eslint-fix ()
+      (interactive)
+      (setq temp-point (point))
```

Comment:
@aaronjensen Why don't you `let`-bind this variable?  It's not a good idea to just `setq` variables this way.